src/display.jl

```diff
@@ -519,15 +529,7 @@ extension.  Allowable extensions are:
 6 times bigger with the same quality!
 
 The `compression` argument controls the compression ratio; `51` is the
-highest compression, and `0` is the lowest (lossless).
-
-When `sleep` is set to `true` (the default), AbstractPlotting will
-display the animation in real-time by sleeping in between frames.
-Thus, a 24-frame, 24-fps recording would take one second to record.
-
-When it is set to `false`, frames are rendered as fast as the backend
-can render them.  Thus, a 24-frame, 24-fps recording would usually
-take much less than one second in GLMakie.
```

Comment:
I removed this bit because I don't see `sleep` being used/passed anywhere. As far as I can tell it just goes as fast as possible by default.

Reply:
Ah, yeah I forgot to clean up the docs when removing that argument